nucliadb_core/src/paragraphs.rs

```diff
@@ -33,9 +33,7 @@ pub struct ParagraphConfig {
 pub struct ParagraphIterator(Box<dyn Iterator<Item = ParagraphItem> + Send>);
 impl ParagraphIterator {
     pub fn new<I>(inner: I) -> ParagraphIterator
-    where
-        I: Iterator<Item = ParagraphItem> + Send + 'static,
-    {
+    where I: Iterator<Item = ParagraphItem> + Send + 'static {
```

Comment:
is this the old formatting rule?

Reply:
I dont know why this changed happened now and not before